spec/unit/content_item_loader_spec.rb

```diff
@@ -1,12 +1,38 @@
 RSpec.describe ContentItemLoader do
   include ContentStoreHelpers
 
+  let(:subject) { described_class.new }
   let!(:item_request) { stub_content_store_has_item("/my-random-item") }
 
-  describe ".load" do
-    it "does not cache calls to the content store" do
-      ContentItemLoader.load("/my-random-item")
-      ContentItemLoader.load("/my-random-item")
+  describe ".for_request" do
+    it "returns a new object per request" do
```

Comment:
Is it worth another test that `it "returns the same object for the same request"` or something?

Reply:
It's very low cost to add / run the test, so yes. I'll add it in.